File: Blog-Project/web/API/EFCore/Author.cs
using System.ComponentModel.DataAnnotations;

namespace API.EFCore
{
    public class Author
    {
        public int Id { get; set; }

        [Required]
        [MaxLength(100)]
        public string FirstName { get; set; }

        [Required]
        [MaxLength(100)]
        public string LastName { get; set; }

        [Required]
        [MaxLength(10)]
        public int Age { get; set; }
    }
}

File: Blog-Project/mobile/blog_app/lib/models/Author.dart
class Author {
  int id = 0;
  String firstName = '';
  String lastName = '';
  int age = 0;

  Author(
      {required this.id,
      required this.firstName,
      required this.lastName,
      required this.age}); // constructor

  // factory
  Author.fromJson(Map<String, dynamic> json) {
    id = json['Id'];
    firstName = json['FirstName'];
    lastName = json['LastName'];
    age = json['Age'];
  }
}
